core/server/models/stripe-customer-subscription.js

```diff
@@ -1,11 +1,48 @@
 const ghostBookshelf = require('./base');
 
+const CURRENCY_SYMBOLS = {
```

Comment:
We are almost striking the "rule of three" with this `CURRENCY_SYMBOLS` variable :sweat_smile:  It's been used in [these](https://github.com/TryGhost/Ghost/blob/b61ccf0889c2e813886fed62a7b95a52926c70c2/core/server/services/members/config.js#L48-L55) two [places](https://github.com/TryGhost/Members/blob/395b2048998c9d43fd37a0f48135dbbdec05aaca/packages/members-api/lib/stripe/index.js#L8-L15).

 I'm starting to think if it's low effort maybe worth moving it down to [StripePaymentProcessor](https://github.com/TryGhost/Members/blob/395b2048998c9d43fd37a0f48135dbbdec05aaca/packages/members-api/lib/stripe/index.js#L17) class's fields and keep it there as an exposed constant?

Reply:
It's now been removed from the the `StripePaymentProcessor` class completely - and it feels like the model/data layer is a better place for it. I'm going to leave it as is for now, until we can work out the best place for it :thinking: